src/core/unit.h

```diff
@@ -515,6 +519,8 @@ typedef struct Unit {
         bool reset_accounting:1;
 
         bool start_limit_hit:1;
+        bool coredump_limit_hit:1;
```

Comment:
Sounds like it would be worth exposing this on dbus too, and emit a signal when it gets set to 'true', so that there's a clear programmatic way to track things hitting the rate limit